test-result-summary-client/src/PerfCompare/PerfCompare.jsx

```diff
@@ -469,11 +474,18 @@ export default class PerfCompare extends Component {
                 curTestScore = parsedVariantsTest.metrics[j].statValues.mean;
                 curTestCI = parsedVariantsTest.metrics[j].statValues.CI;
 
-                // Get the metric's units
-                // get BenchmarkRouter & Metric files from server
-                let parserProps = await getParserProps();
+                // To get the metric's higherbetter/units
+                // first check if Metric does exist in starte, if not get Metric info from server
+                let benchmark = curVariantData["benchmark"]
```

Comment:
Please use `const` for variables that do not change. I am not going to list all of them. Please fix the issue in this PR.